nvflare/private/fed/server/job_runner.py

```diff
@@ -92,23 +92,15 @@ def handle_event(self, event_type: str, fl_ctx: FLContext):
             self.stop()
 
     @staticmethod
-    def _make_deploy_message(job: Job, app_data, app_name):
+    def _make_deploy_message(job: Job, app_data, app_name, fl_ctx):
         message = Message(topic=TrainingTopic.DEPLOY, body=app_data)
         message.set_header(RequestHeader.REQUIRE_AUTHZ, "true")
 
-        message.set_header(RequestHeader.ADMIN_COMMAND, AdminCommandNames.SUBMIT_JOB)
+        message.set_header(RequestHeader.ADMIN_COMMAND, AdminCommandNames.DEPLOY_APP)
```

Comment:
No, we can't change the command name to DEPLOY_APP here. The name SUBMIT_JOB corresponds to the right that the client side will perform with the current fed-authz solution.

Reply:
changed to use SUBMIT_JOB command name.